### PR TITLE
Acrylic page: Scaling fixes

### PR DESCRIPTION
**Pull request by salmanmkc**
So it scales well when on 175% or 125% scaling. This should be merged after the previous PR, since I branched off it to avoid merge conflicts.

## Description
Values won't scale well if using 175% scaling, modified them all.

## Motivation and Context
Fix scaling offsets.

## Screenshots (if appropriate):
Can't really tell, since it makes a difference of a few pixels.
image

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: XamlControlsGallery/ControlPages/AcrylicPage.xaml [hunk 1]
```diff
@@ -112,7 +112,7 @@
             <local:ControlExample.Options>
                 <StackPanel>
                     <TextBlock Text="Tint Opacity :" Margin="0,0,0,12"/>
-                    <Slider x:Name="OpacitySliderInApp" ValueChanged="Slider_ValueChanged" Minimum="0" Maximum="1" Width="200" SmallChange="0.01" StepFrequency="0.01" 
+                    <Slider x:Name="OpacitySliderInApp" ValueChanged="Slider_ValueChanged" Minimum="0" Maximum="1" Width="200" SmallChange="0.0001" StepFrequency="0.0001"
```
Comment (YuliKl) [inline XamlControlsGallery/ControlPages/AcrylicPage.xaml:115]: Looks like this change snuck in from #312

File: XamlControlsGallery/ControlPages/AcrylicPage.xaml [hunk 2]
```diff
@@ -163,7 +163,7 @@
             <local:ControlExample.Options>
                 <StackPanel>
                     <TextBlock Text="Tint Opacity :" Margin="0,0,0,12"/>
-                    <Slider x:Name="OpacitySlider" ValueChanged="Slider_ValueChanged" Minimum="0" Maximum="1" Width="200" SmallChange="0.01" StepFrequency="0.01" 
+                    <Slider x:Name="OpacitySlider" ValueChanged="Slider_ValueChanged" Minimum="0" Maximum="1" Width="200" SmallChange="0.0001" StepFrequency="0.0001"
```
Comment (YuliKl) [inline XamlControlsGallery/ControlPages/AcrylicPage.xaml:166]: This change also looks to be from #312

File: XamlControlsGallery/ControlPages/AcrylicPage.xaml [hunk 3]
```diff
@@ -216,10 +216,10 @@
             <local:ControlExample.Options>
                 <StackPanel>
                     <TextBlock Text="Tint Opacity :" Margin="0,0,0,12"/>
-                    <Slider x:Name="OpacitySliderLumin" ValueChanged="Slider_ValueChanged" Minimum="0" Maximum="1" Width="200" SmallChange="0.01" StepFrequency="0.01" 
+                    <Slider x:Name="OpacitySliderLumin" ValueChanged="Slider_ValueChanged" Minimum="0" Maximum="1" Width="200" SmallChange="0.0001" StepFrequency="0.0001"
```
Comment (YuliKl) [inline XamlControlsGallery/ControlPages/AcrylicPage.xaml:219]: Let's try to keep the changes separate :)

**Comment by salmanmkc** [comment]
Separated now

**Pull request merged**